tests/integration_tests/performance/network_tcp_throughput_test_config.py

```diff
@@ -0,0 +1,359 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Configuration file for the network TCP throughput test."""
+
+DEBUG = False
+
+IPERF3 = "iperf3"
+THROUGHPUT = "throughput"
+THROUGHPUT_TOTAL = "total"
+DURATION = "duration"
+DURATION_TOTAL = "total"
+RETRANSMITS = "retransmits"
+RETRANSMITS_TOTAL = "total"
+CPU_UTILIZATION_HOST = "cpu_utilization_host"
+CPU_UTILIZATION_GUEST = "cpu_utilization_guest"
+BASE_PORT = 5000
+CPU_UTILIZATION_VMM = "vmm"
+CPU_UTILIZATION_VCPUS_TOTAL = "vcpus_total"
+CONFIG = {
+    "time": 20,  # seconds
+    "load_factor": 1,
+    "modes": {
+        "g2h": [""],
+        "h2g": ["-R"],
+        "bd": ["", "-R"]
+    },
+    "protocols": [
+        {
+            "name": "tcp",
+            "omit": 5,
+            "window_size": [16, 256, None],
+            "pkt_size": [None, 1024],
```

Comment:
Please add also a small packet size 128 or 96. The 128K is not the packet size, but the payload size that is later segmented, since nobody is sending a 128K packet on the wire being limited first by mss at tcp level and finaly but MTU. The window size looks good now